gui/webdisplay/src/TWebWindow.cxx

```diff
@@ -326,6 +343,18 @@ std::string ROOT::Experimental::TWebWindow::RelativeAddr(std::shared_ptr<TWebWin
    return res;
 }
 
+///////////////////////////////////////////////////////////////////////////////////
+/// returns connection for specified connection number
+/// Total number of connections can be retrieved with NumConnections() method
+
+unsigned ROOT::Experimental::TWebWindow::GetConnectionId(unsigned num) const
+{
+   for (auto iter = fConn.begin(); iter != fConn.end(); ++iter) {
+      if (num-- == 0) return iter->fConnId;
+   }
+   return 0;
```

Comment:
Really? Or should this `throw` or something?

Reply:
Either one uses exception everywhere or only in very special situations.
This should be convention, which followed everywhere.
I have nothing against exceptions, but this is not the only place then.

Reply:
We use exceptions in the "ROOT 7" interfaces. "return 0" doesn't seem to be an appropriate way to handle this case?